pymc3_hmm/step_methods.py

```diff
@@ -48,25 +52,22 @@ def ffbs_astep(gamma_0: np.ndarray, Gammas: np.ndarray, log_lik: np.ndarray):
     # Number of observations
     N: int = log_lik.shape[-1]
 
-    # Number of states
-    M: int = gamma_0.shape[-1]
-    # assert M == log_lik.shape[-2]
-
     # Initial state probabilities
     gamma_0_normed: np.ndarray = gamma_0.copy()
     gamma_0_normed /= np.sum(gamma_0)
 
     # "Forward" probabilities
-    alphas: np.ndarray = np.empty((M, N), dtype=float)
+    alphas = lik_dict["alphas"]
+
     # Previous forward probability
     alpha_nm1: np.ndarray = gamma_0_normed
 
     # Make sure we have a transition matrix for each element in a state
     # sequence
     Gamma: np.ndarray = np.broadcast_to(Gammas, (N,) + Gammas.shape[-2:])
 
-    lik_n: np.ndarray = np.empty((M,), dtype=float)
-    alpha_n: np.ndarray = np.empty((M,), dtype=float)
+    lik_n = lik_dict["lik_n"]
```

Comment:
The type information for these variables shouldn't change, so it doesn't need to be removed.